### DISCUSSION TITLE
Exclude showing Adwaita-icon-theme in "Appearance" ?

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#2567] [General]
Stumbled upon that reading https://pointieststick.com/2024/05/03/this-week-in-kde-looking-towards-plasma-6-1/

> System Settings no longer lets you choose GNOME’s Adwaita or High Contrast icon themes as your systemwide icon theme, because despite registering themselves as FreeDesktop-compatible icon themes, [they are no longer actually designed to be used this way](https://gitlab.gnome.org/GNOME/adwaita-icon-theme/-/issues/288) and will break everything from KDE if you try anyway (me: Nate Graham, Plasma 6.0.5. [Link](https://bugs.kde.org/show_bug.cgi?id=486409))

In fact  `adwaita-icon-theme` is installed by default together with gtk3/4  and choosing it breaks most of LXQt:

![immagine](https://github.com/lxqt/lxqt/assets/10681413/22de437f-f792-4674-a181-81e694c83fc2)

![immagine](https://github.com/lxqt/lxqt/assets/10681413/bccbb091-c7a2-4967-b9a2-f7465ffaf033)

"High Contrast" instead doesn't look to break icons apparently but maybe I missed something.

**Comment by tsujan** [comment]
IMO, we don't need to hide how nonstandard GNOME is. Moreover, a nonstandard icon theme can easily be made; it's not our job to check it.

**Reply by Gnomasz** [reply]
> a nonstandard icon theme can easily be made

This is true, I have encountered a lot of incomplete icon themes while looking for something that I'd like. However, while the other icon themes needed my active pursuit to be installed, it's hard _not to_ install Adwaita. Would a distro maintainer be able to hide Adwaita or any other preinstalled icon theme from this list?

Keeping Adwaita there isn't necessarily a display of how nonstandard Gnome is. Even a FOSS-aware user who knows that Adwaita comes from Gnome could simply think "this LXQt can't even apply one of the biggest icon themes there is".

**Reply by tsujan** [reply]
> Even a FOSS-aware user who knows that Adwaita comes from Gnome could simply think....

Trying to control what people might think is futile.

GNOME wanted to be different since GNOME3, and now, it has become like Enlightenment: an isolated DE.

**Reply by stefonarch** [reply]
I would be +1 to hide it - it's the same as a bug IMO.

**Reply by tsujan** [reply]
> it's the same as a bug IMO

It is, of course, but not *our* bug.

EDIT:
As a side note, they've called it "Adwaita (The Only One)". I'd interpret it as "Adwaita (The Only Crappy One)"  ;)

**Reply by stefonarch** [reply]
>It is, of course, but not our bug.

If we display a known bug it is also ours :)

**Reply by tsujan** [reply]
IMO, it isn't, especially when it's so remote from us. Let me explain my position with respect to hiding others' bugs.

If there's a bug in Qt, we definitely try to circumvent it until it's fixed in Qt. The same is true for a basic library like GLib as well as other libraries that we use.

But that doesn't mean whenever the devs of a DE, as remote from LXQt as GNOME, do something weird we need to react by changing our codes.

**Reply by Roberalz** [reply]
Personally, I think there are more important things than an icon pack. What is the percentage of use of adwaita in LXQt to dedicate time to it?

You have too much development work to hide bugs from other people.

Thanks!

**Reply by tsujan** [reply]
We could (or perhaps, should) add something like this to our Wiki: "Warning: Unfortunately, Adwaita introduces itself as a standard icon set, although it is not so. If you use it, you will see missing icons everywhere."
